pkgs/openclarity/kubeclarity/registry.yaml

```diff
@@ -0,0 +1,24 @@
+packages:
+  - type: github_release
+    repo_owner: openclarity
+    repo_name: kubeclarity
+    description: KubeClarity is a tool for detection and management of Software Bill Of Materials (SBOM) and vulnerabilities of container images and filesystems
+    asset: kubeclarity-cli-{{trimV .Version}}-{{.OS}}-{{.Arch}}.{{.Format}}
+    format: tar.gz
+    files:
+      - name: kubeclarity
+        src: kubeclarity-cli
```

Comment:
Hmm. According to the document, the command name is `kubeclarity-cli`.

https://github.com/openclarity/kubeclarity#cli

```console
$ kubeclarity-cli analyze --input-type image nginx:latest -o nginx.sbom
```

And the original binary is also `kubeclarity-cli`.

But according to the help message, the command name is `kubeclarity`.

```console
$ kubeclarity-cli --help
KubeClarity content analyzer and vulnerability scanner

Usage:
  kubeclarity [command]

Available Commands:
  analyze     Content analyzer
  completion  Generate the autocompletion script for the specified shell
  help        Help about any command
  scan        Vulnerability scanner

Flags:
      --config string   config file (default is $HOME/.kubeclarity.yaml)
  -h, --help            help for kubeclarity
  -t, --toggle          Help message for toggle
  -v, --version         version for kubeclarity

Use "kubeclarity [command] --help" for more information about a command.
```

Which is correct? 🤔

Reply:
I opened an issue just in case.

- https://github.com/openclarity/kubeclarity/issues/532

Reply:
`kubeclarity-cli` is correct.

- https://github.com/openclarity/kubeclarity/issues/532#issuecomment-1768419388

Reply:
done

Reply:
Thank you!

Reply:
BTW, could you stop force push the next time? 🙏
https://github.com/suzuki-shunsuke/oss-contribution-guide#dont-do-force-pushes-after-opening-pull-requests